plugins/ship_it.plug

```diff
@@ -0,0 +1,12 @@
+[Core]
+name = ship_it
+module = ship_it
+
+[Documentation]
+description = Show motivational ship it squirrel images.
+
+[Python]
+version = 3
+
+[Errbot]
+
```

Comment:
This is autogen from errbot, so it is present in each of them, will remove it everywhere :+1: